src/main/java/Duke.java

```diff
@@ -1,10 +1,223 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args)  {
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+
+        String userInput;
+        Scanner in = new Scanner(System.in);
+        Task[] userTasks = new Task[0];
```

Comment:
Try to avoid magic literals. There has been a few more instances in the rest of your code. You can use a named constant makes the code easier to understand because the name tells us more about the meaning of the number.